src/content/docs/configuration/providers.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Providers
+description: Understand Providers and configure new Targets.
+sidebar:
+  label: Providers
+---
+
+Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
+
+1.  __Provider__  
+    A platform capable of deploying Workspaces on.
+    Examples of Providers may include:
+    *   Container management platforms such as Docker or Podman;
+    *   Cloud hosting providers such as AWS or DigitalOcean.
+2.  __Target__  
+    A set of configuration that governs how Daytona manages and deploys Workspaces.
+    Each Target is tied to an individual Provider.
+    Daytona allows you to configure and switch between multiple Targets at Workspace creation time.
+
+Daytona's architecture decentralizes the role of a Provider into a separate service.
+The [Daytona Server](/reference/server) communicates with Providers to execute operations relating to Workspace creation and lifecycle management.
+
+This makes it possible to create new Providers to govern Workspace management on any platform or hosting providers.
+
+## Managing Targets
+
+### Set a Target
+You can set a new Target for Daytona to use when managing Workspaces.
+
+__Prerequisites__
+*   A working installation of Daytona.
+*   At least one Provider installed.
+
+__Procedure__
+1.  Execute the following command:
+    ```shell
+    daytona target set
+    ```
+2.  Select the appropriate Provider for the environment you want to deploy to.
+3.  Select `New Target`.
+4.  Enter a name for your Target.
+5.  Enter the appropriate configuration options when prompted.
+6.  Click `Enter` to add the Target.
+
+### Remove a Target
+You can remove an existing target.
+Once a Target is removed, you will not be able to manage or deploy Workspaces without adding another Target with the same configuration.
+
+__Prerequisites__
+*   A working installation of Daytona.
+*   At least 1 configured Target.
+
+__Procedure__
+1.  Execute the following command to enter the interactive terminal UI:
+    ```shell
+    daytona target set
+    ```
+2.  Select the target to remove.
+
+## Using the Docker Provider
```

Comment:
I don't agree with having the `Docker Provider` here. I think every provider should have it's own subpage.

Reply:
For now lets not over complicate the docs with submenus. @Tpuljak so if you have other comments feel free to share but lets push this forward and we will see how the docs grow in the future

Reply:
Okay, we can leave this for now, but once another provider hits, we can go with @osslate's suggestion